src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class ColorSupplier {
+    private static final String[] COLORS = {"Pink", "Green", "Sky Blue",
+            "Purple", "Yellow", "Mint"};
+
+    public static String getRandomColor() {
+        Random random = new Random();

```

Comment:
should be class level
https://github.com/mate-academy/jv-oop-advanced/blob/master/checklist.md#think-about-which-variables-should-be-local-in-the-method-and-which-should-be-class-level

Reply:
not fixed

Reply:
not fixed